common/djangoapps/student/models.py

```diff
@@ -1355,6 +1357,9 @@ def update_enrollment(self, mode=None, is_active=None, skip_refund=False):
         if activation_changed:
             if self.is_active:
                 self.emit_event(EVENT_NAME_ENROLLMENT_ACTIVATED)
+                if not is_testing_environment():
+                    emit_course_progress_event(self.user, CourseOverview.get_from_id(self.course_id))
+
```

Comment:
Actually we recently changes our way to handle this kind of change, We write a helper in our adg code and simply call that here and move this test env check in that helper this way only single line will change in core file. For now we can leave it this way. Just mentioning for future reference.